index.html

```diff
@@ -0,0 +1,152 @@
+        <ul class="desktop-menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+      </div>
+    </nav>
+    <section id="header">
+      <div>
+        <p class="blockquote">"Put on your dancing shoes!!!"</p>
+        <h1 class="header-class">International <br> <span>Tango</span> <br> Festival</h1>
+      </div>
+      <div class="introduction">
+        <p><span class="quote">"The most important Tango event in South America"</span></p>
+        <p>The Internatiuonal Tango Festival will be held in Bogota, Colombia. The most famous milongueros from Colombia will present in this magnificent event.</p>
+      </div>
+      <div class="place">
+        <p class="date">2022.11.01 ~ 06 (monday to sunday)</p>
+        <p class="salon">@ Hotel Tequendama, Bogota , Colombia</p>
+      </div>
+    </section>
+
+    <main>
+      <!-- Program Section -->
+      <section id="program">
+        <!-- cross bg -->
+        <div class="cross-bg">
+          <h2 class="program-heading">Main Program</h2>
+          <div class="line"></div>
+
+          <!-- programs-container -->
+          <div class="programs-container">
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_01.png" alt="lecture icon">
+              <h3 class="program-card-heading">Workshop</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_02.png" alt="exhibition icon">
+              <h3 class="program-card-heading">Milonga Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_03.png" alt="forum icon">
+              <h3 class="program-card-heading">Tango Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_04.png" alt="workshop icon">
+              <h3 class="program-card-heading">Vals Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Milonga</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
```

Comment:
For semantic reasons and to improve your site's accessibility, I'd suggest that you use an `article` tag in place of a `div` tag for each of the program card to avoid running into a div soup. Please fix this 👏

Reply:
ok